src/cnc/flavors/aws/ecs/1/build/main.sh.j2

```diff
@@ -38,3 +39,11 @@ push_{{ service.name }}_image_tags
 
 {% block post_build_commands %}
 {% endblock %}
+
+{% endif %}
+
+{% if service.is_serverless %}
+{% block serverless_block %}
+{% include "main_lambda.sh.j2" %}
```

Comment:
this main_lambda file contains alot of logic that is already handled here, so really we can remove this include line and just replace it with
```
build_{{ service.name }}
```